ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ListSourceExecutor.java

```diff
@@ -475,4 +475,12 @@ private SourceDescriptionWithWarnings(
       this.description = description;
     }
   }
+
+  public static String getFullname(final String name) {
+    if (name.contains("\"")
```

Comment:
we should use `IdentifierUtil#needsQuotes` instead of duplicating that logic here